autograder/autograder/execution_environments/container_network.py

```diff
@@ -114,6 +114,10 @@ def create(self, execution_script, arguments, more_than_one):
             self.log_function(f'ERROR: The image {self.image} is not available on this worker')
             client.close()
             raise
+        except Exception:
```

Comment:
Can we catch more specific exceptions here for why the container failed to start?

Reply:
I had similar concerns, but the full exception stack trace is logged further up the chain.

Reply:
Yes, we are intentionally adding a broad exception catch.  We experienced an unexpected network timeout exception on production and we didn't catch that error at a good place.  We don't think a specific exception catch here warranted right now, since we'd ALSO want to add the catch all exception.